rm-datasource/src/test/java/io/seata/rm/datasource/undo/BaseH2Test.java

```diff
@@ -15,23 +15,14 @@
  */
 package io.seata.rm.datasource.undo;
 
-import io.seata.rm.datasource.sql.struct.ColumnMeta;
-import io.seata.rm.datasource.sql.struct.Field;
-import io.seata.rm.datasource.sql.struct.KeyType;
-import io.seata.rm.datasource.sql.struct.Row;
-import io.seata.rm.datasource.sql.struct.TableMeta;
-import io.seata.rm.datasource.sql.struct.TableRecords;
+import io.seata.rm.datasource.sql.struct.*;
```

Comment:
Please import explicit class type.do not use .*

Reply:
ok